### PR TITLE
Adopt FQCN

### PR DESCRIPTION
**Pull request by ssbarnea**
- Use FQCN for community.docker
- Add community.docker to requirements.yml
- Fixes ansible-lint eco job

See https://github.com/ansible-community/ansible-lint/runs/4693742816?check_suite_focus=true

### CODE REVIEW

**Comment by ssbarnea** [comment]
@konstruktoid Please take a look at ^, eco job was broken for more than a month due to this repo.

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -104,8 +104,7 @@ verifier:
   name: ansible
 lint: |
   set -e
-  ansible-lint --exclude .git --exclude .github --exclude tests/
-  yamllint -d '{"extends":"default","ignore":".tox*\n.git*","rules":{"line-length":{"max":120,"level":"warning"}}}' .
+  ansible-lint
```
Comment (konstruktoid) [inline molecule/default/molecule.yml:107]: Why?

**Comment by konstruktoid** [comment]
See https://github.com/konstruktoid/ansible-docker-rootless/commit/3c60a35fc644ed0f01b67ba355537d3a7fe16ee3 and
https://github.com/konstruktoid/ansible-docker-rootless/commit/9bb41e5439b258e3b71d087518f126992bde81c1

Related https://github.com/ansible-community/molecule/issues/3266

**Comment by ssbarnea** [comment]
- I am not aware of `collections.yml` being a documented way to declare dependencies. On the other hand, `requirements.yml` is.
- no need to exclude folders not tracked by git because that is what ansible-lint does by default! i think that feature is more than an year old.
- no need to exclude folders because we want yamllint bit to lint those files (that is why yamllint no longer needs to be called separately)

BTW, ping me on irc if you can, we may be able to address questions faster.

**Comment by konstruktoid** [comment]
> * I am not aware of `collections.yml` being a documented way to declare dependencies. On the other hand, `requirements.yml` is.

I've probably misread https://molecule.readthedocs.io/en/latest/configuration.html#molecule.dependency.ansible_galaxy.AnsibleGalaxy then;
"Use “role-file” if you have roles only. Use the “requirements-file” if you need to install collections."

```yaml
    role-file: requirements.yml
    requirements-file: collections.yml
```

> * no need to exclude folders not tracked by git because that is what ansible-lint does by default! i think that feature is more than an year old.
> * no need to exclude folders because we want yamllint bit to lint those files (that is why yamllint no longer needs to be called separately)

yeah, that should have been handled better

> BTW, ping me on [irc/matrix](https://matrix.to/#/@ssbarnea:matrix.org) if you can, we may be able to address questions faster.

**Pull request merged**